depeche_db/_subscription.py

```diff
@@ -234,14 +238,11 @@ def _calculate_unprocessed_message_counts():
             if not self._max_aggregated_stream_positions_cache:
                 return {}
 
-            max_aggregated_stream_positions = (
-                self._max_aggregated_stream_positions_cache
-            )
             unprocessed_message_counts = []
             for (
                 partition_number,
                 max_position,
-            ) in max_aggregated_stream_positions.items():
+            ) in self._max_aggregated_stream_positions_cache.copy().items():
```

Comment:
Make sure other uses of `_max_aggregated_stream_positions_cache` are also thread-safe!